native/packages/react-native-bpk-component-banner-alert/src/BpkBannerAlert.android.js

```diff
@@ -159,7 +131,6 @@ const BpkBannerAlert = props => {
     dismissButtonLabel,
     toggleExpandedButtonLabel,
     show,
-    style: userStyle,
```

Comment:
Why is this getting removed? This is a breaking change

Reply:
Styles will still be applied to the outer view as part of `{...rest}`